wiki/Geki/en.md

```diff
@@ -9,6 +8,44 @@ tags:
 
 *See also: [Katu](/wiki/Katu)*
 
-**Geki (激)**, or *Elite Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset with the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note. This idea originates from the Nintendo DS game Elite Beat Agents in which the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+**Geki (激)**, or *Elite Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset with the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.
 
-<!-- TODO: Add links-->
+By obtaining a Geki, health bar gain bonus obtained is stronger than just ending a comboset with a non-Geki 300s.
+
+This idea originates from the Nintendo DS game Elite Beat Agents in which the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+
+## Screenshots
+
+In here provided the screenshots of obtaining a Geki/Elite Beat from Elite Beat Agents, Ouendan 2, and osu! itself.
+
+![](img/eba-bornlove-300g.jpg) ![](img/oto-sambomaster-300g.jpg)
+
+![](img/osu-lonelest-300g.jpg)
+
+## Other game modes
+
+### osu!taiko
+
+Geki is only shown on result screen to show all big note hitting succession.
+
+### osu!catch
+
+Geki shows that the completed the comboset without breaking any single combo.
+
+### osu!mania
+
+Geki is to determine a **MAX** or a "320", as the mode calculates this judgment as the base of maximum score instead a 300 itself. There's no accuracy difference between a 300 and a MAX in osu!mania, therefore this judgment gives extra score with same accuracy.
```

Comment:
uhh, this is probably what you wanted to say, right?
```suggestion
Geki is shown as a sprite with rainbow-coloured `MAX` written over it. During the score calculation process, it is worth 320 points instead of 300. There's no accuracy difference between a 300 and a MAX in osu!mania.
```